setup.py

```diff
@@ -0,0 +1,26 @@
+import setuptools
+
+with open("Readme.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
```

Comment:
We could use entrypoints here to install HSTP as a proper command line tool

https://packaging.python.org/en/latest/specifications/entry-points/